main.go

```diff
@@ -193,6 +198,15 @@ func findFiles(patterns []string) (_ []string, err error) {
 	return sortedFiles, err
 }
 
+func modify(filename string, content []byte) error {
+	return ioutil.WriteFile(filename, content, 0644)
+}
+
+func preview(filename string, originalContent []byte, modifiedContent []byte, comment string, stderr io.Writer) error {
```

Comment:
Can we pass the array of comments instead of a string? And then change the representation in this function instead.

The reason this is valuable is that this allows us to modify the representation easily without changing other codes.